coalib/bearlib/__init__.py

```diff
@@ -134,9 +136,11 @@ def deprecate_bear(bear):
     Now let's run the bear:
 
     >>> import sys
-    >>> logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+    >>> from coalib.output.Logging import configure_logging
+    >>> configure_logging(stdout=True,color=False)
+    20
     >>> SomeOldBear().run()
-    WARNING:root:The bear SomeOldBear is deprecated. Use SomeBear instead!
+    [WARNING]... The bear SomeOldBear is deprecated. Use SomeBear instead!
```

Comment:
is the date not also enclosed in square brackets? If so I would improve the example by adding those: `[WARNING][...] text text text`.

But if you boast the line length limit, then nvm ;)